### ISSUE TITLE
Timestamps in comments not working

### ISSUE THREAD

**Issue by swarnenduu** [libre-tube/LibreTube#3062]
### Steps to reproduce

1. Find or write a comment that includes a timestamp e.g. '31:25'
2. Click on the timestamp
![Screenshot_20230213-223646-864](https://user-images.githubusercontent.com/86847643/218524268-49a8e8d9-df2f-4f40-bef9-9b2ffb1b3444.png)

### Expected behavior

The playing video should play from the time mentioned in the timestamp.

### Actual behavior

Video plays as it is. No effect of the timestamp.

### LibreTube version

0.12.0

### Android version

Android 12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Duplicate of #1383

**Comment by swarnenduu** [comment]
> Duplicate of #1383

Yes right. It was a mistake. Was just now going to delete my issue that you marked as duplicate 👍
